pages/docs/reference/whatsnew14.md

```diff
@@ -500,3 +500,20 @@ Complete the initial configuration, and when you add a new dependency to `cocoap
 The new dependency will be added automatically. No additional steps are required.
 
 Learn [how to add dependencies](native/cocoapods.html).
+
+## Kotlin/JS
+
+### New Gradle DSL
+The `kotlin.js` Gradle plugin comes with an adjusted Gradle DSL, which provides a number of new configuration options and is more closely aligned to the DSL used by Kotlin's `multiplatform` plugin. Some of the most impactful changes include:
```

Comment:
In most places, we use the terminology `kotlin-multiplatform` plugin

Reply:
Thanks, changed.